aws_resource_count.py

```diff
@@ -211,7 +259,22 @@ def get_region_cluster_nodes(session: CoveSession, service_name: str, region_nam
         "function": get_region_cluster_nodes,
         "display_name": "Container Hosts",
         "workload_units": 1
-    }
+    },
+    "rds": {
+        "function": get_region_rds_resources,
+        "display_name": "RDS Instances and Clusters",
+        "workload_units": 1
+    },
+    "dynamodb": {
+        "function": get_region_dynamodb_resources,
+        "display_name": "DynamoDB Tables",
+        "workload_units": 50
+    },
+    "redshift": {
+        "function": get_region_redshift_resources,
+        "display_name": "Redshift Clusters",
```

Comment:
We may want to merge the count of DDB + RedShift cause I think we want one value for DataWarehouses